### PR TITLE
Fix use of wordings without domain

### PR DESCRIPTION
**Pull request by eternoendless**

### CODE REVIEW

File: ps_emailsubscription.php [hunk 1]
```diff
@@ -1264,7 +1264,7 @@ public function hookActionDeleteGDPRCustomer($customer)
             if (Db::getInstance()->execute($sql)) {
                 return json_encode(true);
             }
-            return json_encode($this->l('Newsletter subscription: Unable to delete customer using email.'));
+            return json_encode($this->trans('Newsletter subscription: Unable to delete customer using email.', array(), 'Modules.Emailsubscription.Admin'));
```
Comment (LouiseBonnard) [inline ps_emailsubscription.php]: Can we take this opportunity to explicit the wording also? I don't understand if the deletion regards the customer or its email. Also, the uppercase is here useless after the colon. :-)

File: ps_emailsubscription.php [hunk 2]
```diff
@@ -1274,7 +1274,7 @@ public function hookActionExportGDPRData($customer)
             if ($res = Db::getInstance()->ExecuteS($sql)) {
                 return json_encode($res);
             }
-            return json_encode($this->l('Newsletter subscription: Unable to export customer using email.'));
+            return json_encode($this->trans('Newsletter subscription: Unable to export customer using email.', array(), 'Modules.Emailsubscription.Admin'));
```
Comment (LouiseBonnard) [inline ps_emailsubscription.php]: Same here!

File: ps_emailsubscription.php (hunk 1 above)
Comment (LouiseBonnard) [reply ps_emailsubscription.php]: Checked with Alex Darmon, please turn the current wording into _Newsletter subscription: no email to export, this customer has not registered._

File: ps_emailsubscription.php (hunk 2 above)
Comment (LouiseBonnard) [reply ps_emailsubscription.php]: Checked with Alex Darmon, please turn the current wording into _Newsletter subscription: no email to export, this customer has not registered._

**Comment by eternoendless** [comment]
@LouiseBonnard do you approve?

**Pull request merged**